rotator_pxi/README.md

```diff
@@ -0,0 +1,60 @@
+# Software Environment of Rotator PXI Controller
+
+*This is the software environment of rotator PXI controller.*
+
+## Overview
+
+*This Dockerfile tries to have the similar software environment in the rotator real-time controller. The NI PXIe-8840 controller is used. The Xenomai dual kernel is installed in the controller. The ipipe, EtherCAT, and HDF5 are installed as well.*
+
+## Difference with the PXI Controller
+
+*1. When installing the Linux kernel, the default configuration (`make defconfig`) is used instead of the customization of real-time contrroler. To do this, the user needs to use `make menuconfig` and enable the [parameters](doc/linuxConfig.md). The user can also copy the existed `.config` file instead.* \
+*2. Because the docker container does not allow the `reboot`, the installed Linux kernel does not run actually.* \
+*3. When build the Xenomai dual kernel, the user needs to use `make menuconfig` and disable the [parametersDual](doc/linuxConfigDual.md).* \
```

Comment:
Better: While building the Xenomai dual kernel, the user needs to use ..

Reply:
Fixed in commit: ce7f486. Thanks!